### DISCUSSION TITLE
Zuluscsi with Ensoniq EPS 16plus sampler.

### DISCUSSION THREAD

**Discussion by Johnnyd1965** [ZuluSCSI/ZuluSCSI-firmware#19] [Q&A] [answered]
Hi,  I am using the zuluscsi with an Ensoniq EPS 16 plus sampler and it works well with the Ensoniq. 

The only issue I initially had with the zuluscsi is if I named any CD images as CD1.img or CD1.iso they were not recognised. I needed to name them as HD1.hda and they work well.   It may have something to do with the default sector block size of 2048 when named as CD1.img or CD1.iso.

I am actually using the following naming convention
HD20_512 EPS samples V1.hda

it works fine.

**Comment by aperezbios** [comment, marked as answer]
By default, when you name a file with the .iso suffix, and don't explicitly specify the sector size, the ZuluSCSI firmware exposes it as a 2048-sector CD-ROM drive, which is most typical for CD-ROM drives. When the prefix of the file is HD, it's exposed as a 512-byte non-removable device. If you name the file CD1_512.iso, which explicitly defines the sector size as 512 bytes, the firmware will expose the SCSI CD-ROM device as a 512-byte CD-ROM.
